docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -82,6 +87,46 @@ updates before advancing the server to the running state.
 
 Changes to the `serverStartState` property do not affect already started servers.
 
+### Scripts for starting and stopping a managed server
+The [`startServer.sh`](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/domain-lifecycle/startServer.sh) and [`stopServer.sh`](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/domain-lifecycle/stopServer.sh) scripts located in the `kubernetes/samples/scripts/domain-lifecycle` directory can be used to start-up or shut down a specific managed server in a WebLogic Server Domain. These scripts accept below four input parameters.
```

Comment:
(1) There is overlap between the scripts markup doc and each script's built-in usage. I suspect it'd be better for usage/doc maintenance and for self documentation purposes if the information was merged into one place - the usage - and the doc minimized so as only to have high level descriptions of each script.
(2) Since the scripts are technically samples, I suspect the majority of their doc probably should be under content/samples (then domain-lifecycle and the sample doc can cross-reference).

Reply:
I have clarified the script usage and cross referenced it from end-user documentation. I have a high level description of the scripts in domain-lifecycle startup document. Please let me know if I missed something. Thanks.

Reply:
LGTM